client/src/app/shared/models/poll/poll.ts

```diff
@@ -19,9 +19,17 @@ export class Poll extends BaseDecimalModel<Poll> {
     public votescast: number;
     public onehundred_percent_base: PollPercentBase;
     public majority_method: MajorityMethod;
+
+    /**
+     * START is this used anymore?
+     */
     public voted_id: number[];
     public user_has_voted: boolean;
     public user_has_voted_for_delegations: Id[];
+    /**
+     * END
+     */
+
```

Comment:
@FinnStutzenstein I've come across these lines. My question: When is this used and when are the fields filled? I have seen references for `user_has_voted` and `user_has_voted_for_delegations`, but I think these fields have to be filled by the client because they are client-specific fields, right? 
The field `voted_id` is an array of numbers and nowhere used. For me it is unclear in which cases this field should be used. Do you know any cases?